theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-commands.ts

```diff
@@ -17,4 +17,9 @@ export namespace TraceExplorerCommands {
         id: 'trace-explorer:remove-trace',
         label: 'Remove Trace'
     };
+
+    export const OPEN_WITH_TRACE_VIEWER: Command = {
+        id: 'trace-explorer:open-with-trace-viewer',
+        label: 'Open with Trace Viewer'
+    };
```

Comment:
Not sure where to add the comment. Could you please change the priority of the menu entry of that command so that it's lower in the list, preferably last. We did the same change in the [vscode-trace-extension PR 249](https://github.com/eclipse-cdt-cloud/vscode-trace-extension/pull/249) after receiving user feedback that it's too intrusive (i.e. too high up with the common open commands). See image. Not sure how to do that in Theia but it should be possible. 

BTW, I see now why you suggest to change it to `Open in Trace Viewer`. It would be consistent to the `Open in Terminal` command. So, I would be ok to use that, but we should also change it in the vscode-trace-extension.

![image](https://github.com/user-attachments/assets/0bb89190-6616-4a93-ab45-e9cb3f5dda7e)

Reply:
> change the priority of the menu entry of that command so that it's lower in the list, preferably last

easily done

> [use "open in trace viewer"] we should also change it in the vscode-trace-extension

Created an issue, that we can schedule as we see fit: https://github.com/eclipse-cdt-cloud/vscode-trace-extension/issues/275

Reply:
I pushed a new version that should address the comments. The context menu entry should be the last item in the "Navigation" section (top part) of the menu. We can't control the order of new entries that might get added by others, but with the vanilla entries, added by Theia, we should be consistently at the end of the section.

Reply:
One question I have: are you bugged by the name here, the constant being named "OPEN_WITH..." but presenting the command as "open in..."?:

![image](https://github.com/user-attachments/assets/25739ac4-3564-4f58-9d9f-6cb214c23091)

Reply:
If so, I can align everything to use "open in"

Reply:
It's now or never because after we release it the command could be triggered by a third-party extension.

Reply:
Good point - let's do it then

Reply:
done, new version pushed